### PR TITLE
Enhance Lunch Algorithm

### PR DESCRIPTION
**Pull request by jonsongoffwhite**
Use natural language processing, ML and AI in order to create a neural network of all lunch requests ever made to identify the most popular approaches to asking for a potential location for the consumption of noon-time food

### CODE REVIEW

**Review by nanorepublica — APPROVED** [review]
😆 😂

**Review by browniebroke — APPROVED** [review]
So much fake news in this PR, this is bad, bad, bad...

File: scripts/lunch.coffee [hunk 1]
```diff
@@ -1,3 +1,21 @@
...
+    robot.hear /at what restaurant would you like to dine for luncheon today?/i, (res) ->
+        res.send("/poll Where should we go for lunch? :gbk: :pizza: :spaghetti: :curry: :sushi: :taco: :burrito: :ramen: :chicken:")
+    robot.hear /Where food/i, (res) ->
+        res.send("/poll Where should we go for lunch? :gbk: :pizza: :spaghetti: :curry: :sushi: :taco: :burrito: :ramen: :chicken:")
+    robot.hear /where food/i, (res) ->
+        res.send("/poll Where should we go for lunch? :gbk: :pizza: :spaghetti: :curry: :sushi: :taco: :burrito: :ramen: :chicken:")
+    robot.hear /Food/i, (res) ->
+        res.send("/poll Where should we go for lunch? :gbk: :pizza: :spaghetti: :curry: :sushi: :taco: :burrito: :ramen: :chicken:")
+    robot.hear /food/i, (res) ->
+        res.send("/poll Where should we go for lunch? :gbk: :pizza: :spaghetti: :curry: :sushi: :taco: :burrito: :ramen: :chicken:")
```
Comment (browniebroke) [inline scripts/lunch.coffee]: Can you make this DRYer? If I want to add a lunch option I have to edit all the options... I'd rather just go for lunch as I'm hungry.

Comment (jonsongoffwhite) [reply scripts/lunch.coffee]: Yeh sure, check the next commit, should sort out a few things

**Pull request merged**